FAQ.md

```diff
@@ -668,6 +669,21 @@ Compiler ninjas may wonder about the counting of loop-invariant sub-expressions,
 
 [top](#Frequently-Asked-Questions)
 
+## How do I fairly compare the performance to that of my in-house code
+
+It's challenging! Here's a potentially non-exhaustive list of things to check:
+
+* Check the physics — e.g., there are many TTI variations out there, are the two codes using the same one? You wanna ensure the PDEs are the same.
+* Check the discretization — do the two codes use the same spatial order? most importantly, is it the same spatial order in all problem dimensions? carefully optimized codes often use lower order stencils in some dimensions. This significantly impacts performance
+* Did you try tuning the performance of the Devito `Operator`? E.g., on GPUs it is worth giving the `par-tile` option a go.
+* Snapshotting: compression, serialization, asynchronous streaming if running on device... Many legacy codes have these features enabled out-of-the-box. Are you using them?
```

Comment:
I would move this towards the end of the list.

Reply:
why?